tests/vmi_configuration_test.go

```diff
@@ -460,6 +460,62 @@ var _ = Describe("Configurations", func() {
 			})
 		})
 
+		Context("with BIOS bootloader method and no disk", func() {
+			It("should find no bootable device by default", func() {
+				By("Creating a VMI with no disk and an explicit network interface")
+				vmi := tests.NewRandomVMI()
+				tests.AddExplicitPodNetworkInterface(vmi)
+
+				By("Enabling BIOS serial output")
+				vmi.Spec.Domain.Firmware = &v1.Firmware{
+					Bootloader: &v1.Bootloader{
+						BIOS: &v1.BIOS{
+							UseSerial: tests.NewBool(true),
+						},
+					},
+				}
+
+				By("Ensuring network boot is disabled on the network interface")
+				Expect(vmi.Spec.Domain.Devices.Interfaces[0].BootOrder).To(BeNil())
+
+				By("Starting a VirtualMachineInstance")
+				vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Expecting no bootable NIC")
+				_, err = tests.NetBootExpecter(vmi)
```

Comment:
The ntebootexpecter seems to be very specific. It may make sense to not place it in util.go, but that is just a nit.